examples/example-vite-svelte-recs/src/dojo/typescript/models.gen.ts

```
@@ -0,0 +1,193 @@
+  can_move: Boolean;
+}
+export const MovesDefinition = {
+  player: RecsType.BigInt,
+  remaining: RecsType.Number,
+  last_direction: DirectionDefinition,
+  can_move: RecsType.Boolean,
+};
+
+// Type definition for `dojo_starter::models::DirectionsAvailable` struct
+export interface DirectionsAvailable {
+  player: BigInt;
```

Biome

[error] Don't use 'BigInt' as a type.

Use lowercase primitives for consistency.
Safe fix: Use 'bigint' instead


(lint/complexity/noBannedTypes)